docs/source/libraries/robocloud_items/index.rst

```diff
@@ -78,13 +82,14 @@ the suite starts.
     Library    RPA.Robocloud.Items
 
     *** Tasks ***
-    Use variables from Robocloud
-        Set task variables from work item
-        Log   Using variables from workspace ${workspace} for user ${user_id}
-
+    Save variables to Robocloud
+        Add work item file    orders.xlsx
+        Set work item variables    user=Dude    mail=address@company.com
+        Save work item
 
-In the following example the work item is modified locally and then saved
-back to Robocloud.
+Later in the process inside a different robot, we can use previously saved
```

Comment:
Do work items have any sort of namespacing / should you be careful to not collide names if you are running many robots in parallel?

Reply:
Currently the work item data payload is just any JSON and the robot can freely modify it. It is however reserved for the execution and there will be no parallel access. Namespacing might or might not come when we start pushing reusable/shareable robots.

Reply:
Okay clear enough. Then the documentation from this PR should be sufficient